src/index.spec.ts

```diff
@@ -219,7 +218,7 @@ describe('EppoJSClient E2E test', () => {
                   {
                     attribute: md5Hash('appVersion'),
                     operator: md5Hash('GT'),
-                    value: encode('10'),
+                    value: base64Encode('10'),
```

Comment:
We had a lingering `encode` in our test, which since its run in the node env, doesn't need `universal-base64`, so I just changed the method and axed the dependency.